src/wast-parser.cc

```diff
@@ -606,16 +606,26 @@ bool WastParser::ParseAlignOpt(uint32_t* out_align) {
   }
 }
 
-Result WastParser::ParseLimits(Limits* out_limits) {
+Result WastParser::ParseLimits(Limits* out_limits, Sharing allowed) {
   WABT_TRACE(ParseLimits);
+  Token sharing_tok;
+  if (PeekMatch(TokenType::Shared)) {
```

Comment:
Should be `"(" "shared" min max ")"` or `min max?`

Reply:
Oh, so an inner set of parens. got it.